metrics/gauge_float64.go

```diff
@@ -2,7 +2,7 @@ package metrics
 
 import "sync"
 
-// GaugeFloat64s hold a float64 value that can be set arbitrarily.
+// GaugeFloat64 methods hold a float64 value that can be set arbitrarily.
```

Comment:
Please don't add `methods` all over, it's a simple interface.

Reply:
Thank you - updated.